src/openforms/config/data.py

```diff
@@ -3,7 +3,9 @@
 
 from django.contrib.admin.templatetags.admin_list import _boolean_icon
 
-Action: TypeAlias = tuple[str, str]
+from openforms.typing import _StrOrPromise
```

Comment:
I think I redefined `StrOrPromise` explicitly, without the `_`

Reply:
right - why do we have both? :P

Reply:
`_StrOrPromise` is defined as a private alias in the django stubs but preferred to export it in a more explicit way